lib/plug/parsers.ex

```diff
@@ -200,23 +200,21 @@ defmodule Plug.Parsers do
       nil ->
         %{conn | body_params: %{}}
     end
+    merge_params(conn)
   end
 
   def call(%Conn{body_params: %Plug.Conn.Unfetched{}} = conn, _opts) do
```

Comment:
This function clause is currently required for [this test](https://github.com/elixir-lang/plug/blob/master/test/plug/parsers_test.exs#L50) to pass. The parsers set `body_params` to `%{}` for non-POST requests.